test/functional/test_site/expected/testAnchorGeneration.html

```diff
@@ -1,96 +1,85 @@
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Anchor Generation Test</title>
-  <link rel="stylesheet" href="markbind/css/bootstrap.min.css">
-  <link rel="stylesheet" href="markbind/css/bootstrap-vue.min.css">
-  <link rel="stylesheet" href="markbind/fontawesome/css/all.min.css">
-  <link rel="stylesheet" href="markbind/glyphicons/css/bootstrap-glyphicons.min.css">
-  <link rel="stylesheet" href="markbind/css/octicons.css">
-  <link rel="stylesheet" href="markbind/css/github.min.css">
-  <link rel="stylesheet" href="markbind/css/markbind.css">
-  <link rel="stylesheet" href="STYLESHEET_LINK">
-  <link rel="stylesheet" href="markbind/layouts/default/styles.css">
-  <meta name="default-head-bottom">
-  <link rel="icon" href="/test_site/favicon.png">
-</head>
-
-<body>
-  <div id="app">
+    <meta name="default-head-top">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="generator" content="MarkBind 2.12.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Anchor Generation Test</title>
+    <link rel="stylesheet" href="markbind/css/bootstrap.min.css">
+    <link rel="stylesheet" href="markbind/css/bootstrap-vue.min.css">
+    <link rel="stylesheet" href="markbind/fontawesome/css/all.min.css">
+    <link rel="stylesheet" href="markbind/glyphicons/css/bootstrap-glyphicons.min.css">
+    <link rel="stylesheet" href="markbind/css/octicons.css">
+    <link rel="stylesheet" href="markbind/css/github.min.css">
+    <link rel="stylesheet" href="markbind/css/markbind.css"><link rel="stylesheet" href="STYLESHEET_LINK"><link rel="stylesheet" href="markbind/layouts/default/styles.css"><meta name="default-head-bottom"><link rel="icon" href="/test_site/favicon.png"></head>
+<body >
+<div id="app">
     <div id="flex-body">
-      <div id="content-wrapper">
-        <h1 id="root-file">Root file<a class="fa fa-anchor" href="#root-file"></a></h1>
-        <h1 id="should-have-anchor">should have anchor<a class="fa fa-anchor" href="#should-have-anchor"></a></h1>
-        <h2 id="should-have-anchor-2">should have anchor<a class="fa fa-anchor" href="#should-have-anchor-2"></a></h2>
-        <h3 id="should-have-anchor-3">should have anchor<a class="fa fa-anchor" href="#should-have-anchor-3"></a></h3>
-        <h4 id="should-have-anchor-4">should have anchor<a class="fa fa-anchor" href="#should-have-anchor-4"></a></h4>
-        <h5 id="should-have-anchor-5">should have anchor<a class="fa fa-anchor" href="#should-have-anchor-5"></a></h5>
-        <h6 id="should-have-anchor-6">should have anchor<a class="fa fa-anchor" href="#should-have-anchor-6"></a></h6>
-        <panel id="should-have-anchor-7"><template slot="_header"><h4 id="should-have-anchor-7">should have anchor<a class="fa fa-anchor" href="#should-have-anchor-7"></a></h4></template> Lorem ipsum
-        </panel>
-        <panel><template slot="_header"><p>Collapsed</p></template>
```

Comment:
Not specifically to do with the head, some other areas lost their indent as well.

Reply:
Fixed it. The option hadn't been updated to `htmlBeautify` from `jsBeautify`.